docker-compose/tmc-docker-compose.yml

```diff
@@ -394,66 +297,24 @@ services:
       - dishleafnode4
       - sdpsubarrayleafnode
       - cspsubarrayleafnode
-#      - add-devs-and-props
+      - tmc-dsconfig
     network_mode: ${NETWORK_MODE}
     container_name: ${CONTAINER_NAME_PREFIX}subarraynode2
     environment:
     - TANGO_HOST=${TANGO_HOST}
     command: >
       sh -c "wait-for-it.sh ${TANGO_HOST} --timeout=30 --strict --
              sudo ln -sf /var/run/rsyslog/dev/log /dev/log &&\
-             tango_admin --ping-device mid_d0001/elt/master 10 &&\
-             tango_admin --ping-device mid_d0002/elt/master 10 &&\
-             tango_admin --ping-device mid_d0003/elt/master 10 &&\
-             tango_admin --ping-device mid_d0004/elt/master 10 &&\
-             tango_admin --ping-device mid_csp/elt/subarray_01 10 &&\
-             tango_admin --ping-device mid_sdp/elt/subarray_1 10 &&\
              tango_admin --ping-device ska_mid/tm_leaf_node/d0001 10 &&\
              tango_admin --ping-device ska_mid/tm_leaf_node/d0002 10 &&\
              tango_admin --ping-device ska_mid/tm_leaf_node/d0003 10 &&\
              tango_admin --ping-device ska_mid/tm_leaf_node/d0004 10 &&\
              tango_admin --ping-device ska_mid/tm_leaf_node/csp_subarray01 10 &&\
              tango_admin --ping-device ska_mid/tm_leaf_node/sdp_subarray01 10 &&\
-             /venv/bin/python /app/tmcprototype/SubarrayNode/SubarrayNode/SubarrayNode.py sa2"
+             /venv/bin/python -u /app/tmcprototype/SubarrayNode/SubarrayNode/SubarrayNode.py sa2 v5"
```

Comment:
Starting with DEBUG level logs will be too much to handle. If may also make it hard to read functional print statements with DEBUG statements.

Reply:
We have changed this to be able to read the docker logs. This is required for now for the testing purposes. We can remove it once the testing is complete.

Reply:
Commit ID: [4fe0b933c9bfe4e5c66c31e0dfae0261bf485831]